testcases/PowerNVDump.py

```diff
@@ -121,9 +129,11 @@ def setUp(self):
             self.c.run_command("cp /etc/kdump.conf /etc/kdump.conf_bck")
         elif 'SLES' in res[0] or 'SLES' in res[1]:
             self.distro = 'sles'
-            self.c.run_command("cp /etc/sysconfig/kdump /etc/sysconfig/kdump_bck")
+            self.c.run_command(
+                "cp /etc/sysconfig/kdump /etc/sysconfig/kdump_bck")
         else:
-            raise self.skipTest("Test currently supported only on ubuntu, sles and rhel")
+            raise self.skipTest(
+                "Test currently supported only on ubuntu, sles and rhel")
```

Comment:
Seems like this patch combines pep8 fixes as well as password less kdump. 
Can you separate them out in 2 different commits?